west.yml

```diff
@@ -363,6 +363,10 @@ manifest:
     - name: zcbor
       revision: 9b07780aca6fb21f82a241ba386ad9b379809337
       path: modules/lib/zcbor
+    - name: cyw43-driver
+      path: modules/cyw43-driver
+      revision: main
+      url: https://github.com/flyfish30/zephyr-cyw43-driver.git
```

Comment:
This contains source code with a license that is not compatible with Zephyr, so you must first delete it. 

https://github.com/georgerobotics/cyw43-driver/blob/1d2227bca200e13c1d6a630032d5f6d7fca69fef/LICENSE.RP

The current implementation of PicoW has resolved this issue, so I think we will follow that.

Reply:
In this license file, this source code can be only used in pico or pico2 board。This patch is used in the WiFi driver of pico2 w, so this source code can be used here, Is this correct?

> Raspberry Pi Ltd (Licensor) hereby grants to you a non-exclusive license to
> use this software solely with the Licensor's microcontroller chip (RP2040) or
> any other semiconductor device produced by the Licensor. No other use is
> permitted under the terms of this licence.
> 

I had use latest source code of main branch of zephyr, the wifi is still not working well in pico2 w board.

Reply:
> In this license file, this source code can be only used in pico or pico2 board。This patch is used in the WiFi driver of pico2 w, so this source code can be used here, Is this correct?

Incorrect.
Codes that provide with zephyr distribution are not allow to change the license conditions by depending on the use.
Zephyr's code must be made available for derivative use on any platform.
Even if the one is competiate with original.
This condition does not meet that requirement.